src/components/json-file-upload/JsonFileUpload.tsx

```diff
@@ -147,7 +148,18 @@ export const JsonFileUpload = ({
             dropzoneProps={{
               accept: ".json",
             }}
-          />
+            hideDefaultPreview
+          >
+            <CodeEditor
+              isLineNumbersVisible
+              code={fileUpload.value as string}
```

Comment:
This is already always a string according to the type, so this type cast can be removed.
```suggestion
              code={fileUpload.value}
```